src/index.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import App from './components/App';
+import rootReducer from './reducers';
 
+const store = createStore(rootReducer);
 ReactDOM.render(
-  <React.StrictMode>
+  <Provider store={store}>
     <App />
-  </React.StrictMode>,
+  </Provider>,
```

Comment:
- [ ] It wraps `App` in `Provider` and assign the whole thing into a variable (Right now you are only rendering the components, but Microverse requires you to wrap them and save them to a variable)

```javascript
const wrappedApp = (
  <Provider store={store}>
    <App />
  </Provider>
);
```